osquery/logger/plugins/kafka_producer.h

```diff
@@ -0,0 +1,129 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#pragma once
+
+#include <atomic>
+#include <future>
+#include <memory>
+#include <mutex>
+
+#include <librdkafka/rdkafka.h>
+
+#include <osquery/core.h>
+#include <osquery/dispatcher.h>
+#include <osquery/logger.h>
```

Comment:
If the header does not need the logger, can you move this to the implementation file?